### ISSUE TITLE
Add accessibility resources to guides

### ISSUE THREAD

**Issue by entcheva** [thoughtbot/guides#546]
We have a wealth of knowledge & resources on accessibility, but it's not documented anywhere for easy access (that I know of). Should we add them to the guides?

Some go-to resources on a11y:
https://www.w3.org/WAI/standards-guidelines/wcag/
https://www.w3.org/WAI/WCAG21/quickref/
https://a11yproject.com/
https://www.accesslint.com/
https://www.deque.com/axe/
https://accessibilityinsights.io/
https://shop.smashingmagazine.com/products/inclusive-design-patterns
https://abookapart.com/products/accessibility-for-everyone
https://css-tricks.com/tag/accessibility/

**Comment by mike-burns** [comment]
Should we add a new top-level `accessibility/` directory? Or would this go under `best-practices/`?

**Comment by tysongach** [comment]
I found these helpful recently:

- https://developer.paciellogroup.com/blog/2015/01/basic-screen-reader-commands-for-accessibility-testing/
- https://www.youtube.com/watch?v=qDm7GiKra28

**Comment by tysongach** [comment]
Oh, and our @thoughtbot/accessibility team here on GitHub.

**Comment by entcheva** [comment]
@mike-burns I think accessibility guidelines should be part of `/best-practices`, organized under each appropriate topic. Thoughts?

I'm working on a PR to add guidelines under `/best-practices#html`.

**Comment by mike-burns** [comment]
+1 `best-practices`.

**Comment by ericwbailey** [comment]
@entcheva I have a PR here for testing and auditing: https://github.com/thoughtbot/guides/pull/579

Maybe this is a case of taking the quick checks section and moving it to best practices? We could add a resources section to either for the links that have been collected in this Issue, as well.

**Comment by entcheva** [comment]
@ericwbailey Awesome, thanks so much for that PR! 

I like your idea of moving the quick checks section to `/best-practices` - IMO it weaves accessibility into the rest of our practice better than a separate section.

**Comment by WilHall** [comment]
Closing this since #579 added an accessibility guide and the direction of #610 defines where it should live.
